libkbfs/key_manager.go

```diff
@@ -119,6 +119,31 @@ func (km *KeyManagerStandard) getTLFCryptKey(ctx context.Context,
 		return kbfscrypto.TLFCryptKey{}, err
 	}
 
+	// Team TLF keys come from the service.
+	if tlfID.Type() == tlf.SingleTeam {
+		tid, err := kmd.GetTlfHandle().FirstResolvedWriter().AsTeam()
+		if err != nil {
+			return kbfscrypto.TLFCryptKey{}, err
+		}
+		keys, _, err := km.config.KBPKI().GetTeamTLFCryptKeys(ctx, tid)
```

Comment:
Yep, we will be, in `KeybaseServiceBase`, once there's a service RPC we can call.

Or are you saying you want to put each of the individual keys into the key cache right away?  I'm not sure that's useful since we may not be using them, and they could evict other keys.